### PR TITLE
changed by Asadullokh && Ibrohim (support) #59 closed

### LINKED ISSUE #59: `Searching` : Create `search bar` for `Authors`

### PR DESCRIPTION
**Pull request by Asadullokh27**
#59 closed

### CODE REVIEW

File: src/BookStore.Presentation/MVC/Controllers/AuthorsController.cs [hunk 1]
```diff
@@ -57,6 +58,20 @@ public async Task<IActionResult> UpdateAsync(Author author)
             return View("Details", updatedAuthor);
         }
 
+        public async Task<IActionResult> SearchAsync(string any)
+        {
+            Author aut;
+            if (int.TryParse(any, out int id))
+            {
+                aut = await _mediator.Send(new GetAuthorByIdQuery() { Id = id });
+                goto res;
```
Comment (Tohirjon-Odilov) [inline src/BookStore.Presentation/MVC/Controllers/AuthorsController.cs:67]: Goto'da foydalanish yaxshimas o'zilarga qarelar

Comment (Tohirjon-Odilov) [reply src/BookStore.Presentation/MVC/Controllers/AuthorsController.cs:67]: Umuman yaxshi ish emas

**Review by Tohirjon-Odilov — CHANGES_REQUESTED** [review]

File: src/BookStore.Presentation/MVC/Controllers/AuthorsController.cs (hunk 1 above)
Comment (SardorSohinazarov) [reply src/BookStore.Presentation/MVC/Controllers/AuthorsController.cs:67]: +

Comment (SardorSohinazarov) [reply src/BookStore.Presentation/MVC/Controllers/AuthorsController.cs:67]: Goto ni almashtirish kerak

Comment (Asadullokh27) [reply src/BookStore.Presentation/MVC/Controllers/AuthorsController.cs:67]: Hop boladi ustoz!

**Pull request merged**